### PR TITLE
CDAP-3155 Add MapR distro support for 4.1 in the installation guide.

### PR DESCRIPTION
**Pull request by henry-cask**
It is officially called 4.1 instead of 4.1.0

### CODE REVIEW

File: cdap-docs/admin-manual/source/installation/installation.rst [hunk 1]
```diff
@@ -152,22 +152,32 @@ For a distributed enterprise, you must install these Hadoop components:
...
 +               +-------------------+-----------------------------------------------------+
 |               | CDH or HDP        | (CDH) 5.0.0 through 5.4.4 or (HDP) 2.0, 2.1, or 2.2 |
++               +-------------------+-----------------------------------------------------+
+|               | MapR              | 4.1                                                 |
 +---------------+-------------------+-----------------------------------------------------+
 | **HBase**     | Apache            | 0.96.x, 0.98.x, and 1.0.x                           |
 +               +-------------------+-----------------------------------------------------+
 |               | CDH or HDP        | (CDH) 5.0.0 through 5.4.4 or (HDP) 2.0, 2.1, or 2.2 |
++               +-------------------+-----------------------------------------------------+
+|               | MapR              | 4.1                                                 |
```
Comment (ghelmling) [inline cdap-docs/admin-manual/source/installation/installation.rst]: I think we should call out "(with HBase)" here.  We don't support MapR-DB yet.

Comment (henry-cask) [inline cdap-docs/admin-manual/source/installation/installation.rst]: Make sense will update the doc with explicit comment.

**Comment by henry-cask** [comment]
Thanks @ghelmling could you help review it again?

**Comment by ghelmling** [comment]
lgtm

**Pull request merged**